static/css/styles.css

```diff
@@ -8,33 +8,33 @@
     --united-violet: 139, 0, 255;
 }
 
+/* General Reset */
 * {
     margin: 0;
     padding: 0;
     box-sizing: border-box;
-    font-family: Arial, sans-serif;
     text-decoration: none !important;
 }
 
+/* Base Font Family */
 body,
 html {
+    font-family: 'Lato', sans-serif;

```

Comment:
I don't see Lato imported anywhere. If you change a global font, make sure it works everywhere..

Reply:
I wasn't sure about this...

I had the import code from {googlefonts I think} then removed it.

Should I reinstate the import on the basis that lato is needed [according to our README] for all secondary text?